examples/pkam_authenticate/src/main.c

```diff
@@ -0,0 +1,68 @@
+#include <stdio.h>
+#include <atclient/atclient.h>
+#include <atclient/atkeysfile.h>
+#include <atclient/atlogger.h>
+
+#define ROOT_HOST "root.atsign.org"
+#define ROOT_PORT 64
+
+#define ATKEYSFILE_PATH "/Users/jeremytubongbanua/.atsign/keys/@smoothalligator_key.atKeys"
```

Comment:
Careful of hard-coded values maybe add a define.h (gitignored) and a define-template.h (with empty values) or however you want to do it

Reply:
Meh, don't let it stop this PR from being merged. But consider changing in the future

Reply:
Ok